data/features.yml

```
@@ -352,6 +422,8 @@ user_authentication:
   product_team: repo_management
```

This is actually Cloud SaaS who owns User auth - though I'm not sure exactly how to name them in this file to avoid breaking things.

---

```suggestion
  product_team: cloud_saas
```

This should do the trick. We'll get an error message if I get it wrong.